pkg/ssh/helper.go

```diff
@@ -90,13 +90,22 @@ func ConfigFromKeyBytes(keyBytes []byte) (*ssh.ClientConfig, error) {
 	return clientConfig, nil
 }
 
-func Run(ctx context.Context, client *ssh.Client, command string, stdin io.Reader, stdout io.Writer, stderr io.Writer) error {
+func Run(ctx context.Context, client *ssh.Client, command string, stdin io.Reader, stdout io.Writer, stderr io.Writer, envVars *map[string]string) error {
```

Comment:
why does this have to be a pointer? The default value of `map[string]string` is already `nil`

Reply:
corrected